bin/proof-generator/include/nil/proof-generator/arg_parser.hpp

```diff
@@ -35,6 +35,7 @@ namespace nil {
 
         struct prover_options {
             boost::filesystem::path proof_file_path = "proof.bin";
+            boost::filesystem::path preprocessed_common_data_path = "preprocessed_common_data.dat";
```

Comment:
Maybe this can have extension .bin? It's written the same way as proof.bin, throgh a marshalling.

Reply:
We've already discussed it with verifier team
https://github.com/NilFoundation/evm-placeholder-verification/issues/105